chat.py

```diff
@@ -12,23 +16,37 @@ def format_system_msg(msg):
     return f'[SYSTEM]: <font color="#ff7f00">{msg}</font>'
 
 # import cmds
+class ChatConfig:
+    """Config stuff for chats."""
+    def __init__(self, config):
+        self.whitelisted = config["whitelisted"]
+        self.banned = config["blacklisted"]
+        self.can_send = config["canSend"]
+        self.locked = config["locked"]
+        self.backups = [0, 0]
+        self.last_message =  datetime.now()
+
+    def dummy_method(self):
```

Comment:
I feel like there is a way to get pylint to not complain about these without adding in placeholder methods, ill look into this.

Reply:
I might know how

Reply:
well I found the way, it's called using a `dataclass`

Reply:
worked